packages/lesswrong/themes/stylePiping.js

```diff
@@ -238,11 +241,14 @@ export const ckEditorStyles = theme => {
         },
         '& .ck-annotation__info-name, & .ck-annotation__info-time, & .ck-comment__input, & .ck-thread__comment-count, & .ck-annotation__main p, & .ck-annotation__info-name, & .ck-annotation__info-time, & .ck-presence-list__counter, &.ck-presence-list': {
           ...commentBodyStyles(theme),
+          marginTop: -10,
```

Comment:
Negative margins are generally concerning. Is there another margin that this is cancelling out, which could be removed instead?